buildserver/proxy_test.go

```diff
@@ -131,92 +131,6 @@ func TestProxy(t *testing.T) {
 				"a_test.go:1:46": "A int",
 			},
 		},
-		"go subdirectory in repo": {
```

Comment:
This removes a test for what seems to be a currently unused feature: where the root URI is set to a subdirectory in a repository. The Go extension always sets the root URI to the top level of the repository. Does anyone know of a use case for this? Optimization? Perhaps this can be brought back when the Go extension starts making use of it.

Reply:
this was part of the original design by @sqs. I don't think we ever used this feature.

Reply:
That sounds like a really good feature to have though that can be supported with the new root URIs

Reply:
In the code nav sync today, we decided that we don't need to support this right now. We can bring this back later without incurring tech debt in the meantime.